pennylane/templates/subroutines/out_adder.py

```diff
@@ -150,12 +179,13 @@ def compute_decomposition(
         x_wires, y_wires, output_wires, mod, work_wires
     ):  # pylint: disable=arguments-differ
         r"""Representation of the operator as a product of other operators.
+
         Args:
             x_wires (Sequence[int]): the wires that store the integer :math:`x`
             y_wires (Sequence[int]): the wires that store the integer :math:`y`
             output_wires (Sequence[int]): the wires that store the addition result
-            mod (int): the modulus for performing the addition, default value is :math:`2^{\text{len(output\_wires)}}`
-            work_wires (Sequence[int]): the auxiliary wires to use for the addition
+            mod (int): the modulo for performing the addition. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`
+            work_wires (Sequence[int]): the auxiliary wires to use for the addition, default is ``None``
```

Comment:
```suggestion
            work_wires (Sequence[int]): the auxiliary wires to use for the addition. The
            work wires are not needed if `mod`=:math:`2^{len(output_wires)}`, otherwise two work wires
            should be provided. Defaults to ``None``.
```